sys/net/wininet.d

```diff
@@ -137,6 +138,11 @@ protected:
 	}
 
 	final static void doDownload(HNet hUrl, void delegate(ubyte[]) sink)
+	{
+        doDownload(hUrl, cast(void delegate(const ubyte[])) sink);
```

Comment:
ae uses tabs for indentation.

Reply:
If you add an `.editorconfig`, it avoids problems with dumb people like myself.